### PR TITLE
Check unmodified PluginManager in "startTest"

### PR DESCRIPTION
**Pull request by stefanbirkner**
Increase test coverage.

### CODE REVIEW

File: plugin-management-library/src/test/java/io/jenkins/tools/pluginmanager/impl/PluginManagerTest.java [hunk 1]
```diff
@@ -76,26 +73,7 @@ public void startTest() {
...
-        doReturn(new HashMap<>()).when(pluginManagerSpy).findPluginsAndDependencies(anyList());
-        doReturn(new ArrayList<>()).when(pluginManagerSpy).findPluginsToDownload(anyMap());
-        doReturn(new HashMap<>()).when(pluginManagerSpy).findEffectivePlugins(anyList());
-        doNothing().when(pluginManagerSpy).listPlugins();
-        doNothing().when(pluginManagerSpy).showSpecificSecurityWarnings(anyList());
-        doNothing().when(pluginManagerSpy).checkVersionCompatibility(any(), anyList());
-        doNothing().when(pluginManagerSpy).downloadPlugins(anyList());
-
-        pluginManagerSpy.start();
+        pluginManager.start();
```
Comment (timja) [inline plugin-management-library/src/test/java/io/jenkins/tools/pluginmanager/impl/PluginManagerTest.java:76]: will this cause the test to be reliant on the network and the update center being available?

Comment (stefanbirkner) [reply plugin-management-library/src/test/java/io/jenkins/tools/pluginmanager/impl/PluginManagerTest.java:76]: You're right. I try to improve the test.

Comment (oleg-nenashev) [reply plugin-management-library/src/test/java/io/jenkins/tools/pluginmanager/impl/PluginManagerTest.java:76]: @stefanbirkner Hi, are you still interested in finishing this pull request?

**Comment by MarkEWaite** [comment]
Closing due to no response in many months

**Pull request closed without merging**